css/instant-articles-wizard.css

```diff
@@ -350,18 +350,27 @@ a.instant-articles-button {
 	font-weight: bold;
 }
 .instant-articles-advanced-settings {
```

Comment:
Selector now targets a <p> instead of an <a> tag so `text-decoration` and `display` properties aren't necessary.